(DOCSP-11896) Fixing next-gen validation issues

JuliaMongo opened the pull request:
Here is the [build](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?jobId=5ffd0c28c5bd3658da4eeb95). 
[DOCSP-13596](https://jira.mongodb.org/browse/DOCSP-13596)

There is one item on which I need to know what to do:
ERROR(includes/styles/corrections.rst:0ish): Unknown directive type "raw".
Can I remove the entire directive?

The second error is also not clear to me, because I have already added an alt there, but the build still complains. It might be a Snoopy error, and if so I will log a ticket for them.
![image](https://user-images.githubusercontent.com/72459727/104262612-f25bfc00-5455-11eb-9fa8-fa075e89c9c8.png)

jeff-allen-mongo commented:
Hi @JuliaMongo, thank you very much for your help with this.

1. For that error, we just need to remove `.. include:: /includes/styles/corrections.rst` from line 9 on `index.txt`
2. For that error, we need to add an `:alt:` to the `run-compass-as-user.png` figure near line 177 of `source/includes/steps-starting-compass-individual-fields.yaml`

Other than that, LGTM!

JuliaMongo commented:
@jeff-allen-mongo  I made the remaining changes and amended my commit. Here is the latest [build](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?jobId=5ffdf51dc5bd3658da78b0ba) and it is clean.

Changes I made this time:
- Added one remaining alt in `source/includes/steps-starting-compass-individual-fields.yaml`.
- Removed a references to `corrections.rst` from `install.txt.
- Removed the `corrections.rst `file itself.
Please approve?

jeff-allen-mongo reviewed (approved):
LGTM! We can merge this in and then backport 🙂
